hybrid-iam-ra-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+exec >&2
+set -eux -o pipefail
+
+declare -r USER_DATA="/.bottlerocket/host-containers/current/user-data"
+declare -r SECRETS_DIR="/.bottlerocket/rootfs/var/lib/eks-hybrid/roles-anywhere/pki/"
```

Comment:
Does this path have a purpose?

All else being equal, it'd be better to write these files to a location that is marked `secret_t`, like `/.bottlerocket/rootfs/root/.aws`. That will block reads from unprivileged containers.

Though, it requires the bootstrap container to run "always" rather than "once", since that's a tmpfs mount that doesn't persist across reboots.